packages/pharos/src/components/image-card/pharos-image-card.wc.stories.mdx

```diff
@@ -177,6 +177,70 @@ export const Selectable = () => html`<pharos-layout tag="ol" style="margin: 1rem
   </li>
 </pharos-layout>`;
 
+export const SubtleSelectable = () => html`<pharos-layout tag="ol" style="margin: 1rem 0">
```

Comment:
This appears to be an exact copy of the existing *Selectable* story which already includes the subtle selectable card—might we not need this?

Reply:
It's a little different in that this block has the `subtle="true"` on all the card elements, whereas the `Selectable` story does not. But I think that reinforces the confusion over the # of variations we have here and the difficulty supporting them.

<img width="1494" alt="image" src="https://user-images.githubusercontent.com/13315416/191798040-66115677-ad46-4fcf-a8b0-4c288a43c26c.png">